web/src/lib/components/shared-components/settings/setting-input-field.svelte

```diff
@@ -62,9 +62,9 @@
 
   onMount(() => {
     if (autofocus) {
+      // tick() should be enough, but svelte is being weird
```

Comment:
We can remove this comment :D

Reply:
was thinking somebody might stumble across it after svelte fixing it, and clean it.

by the way, I replaced the `?.` operator, and the catching, they looked like they might be hiding errors